vizro-core/src/vizro/static/css/layout.css

```diff
@@ -170,17 +171,10 @@
   color: var(--text-active);
 }
 
-.collapse-button-tooltip {
-  height: 24px;
-  left: -12px;
-  position: relative;
-  top: 36px;
-}
-
```

Comment:
@nadijagraca - I've removed this CSS className because it didn't seem to affect anything? Could you double-check for me if that's fine to do or if there was a reason why we had this class in.

Reply:
We had this class to position the tooltip, but it does seem it has been fixed with the latest changes so it is safe to remove.